samples/apps/bookinfo/src/mysql/mysqldb-init.sql

```diff
@@ -10,4 +10,4 @@ CREATE TABLE `ratings` (
   PRIMARY KEY (`ReviewID`)
 );
 INSERT INTO ratings (ReviewID, Rating) VALUES (1, 5);
-INSERT INTO ratings (ReviewID, Rating) VALUES (2, 6);
+INSERT INTO ratings (ReviewID, Rating) VALUES (2, 4);
```

Comment:
is 6 breaking the UI in some way ?

Reply:
Yes, the UI only shows up to 5 "empty" stars. So, while the UI can display more than five "filled" stars, it would exceed the maximum a user would expect.

Reply:
Yes, it looks pretty bad. May be use blue stars as next iteration?